controllers/osccluster_securitygroup_controller.go

```diff
@@ -206,7 +206,7 @@ func reconcileSecurityGroupRule(ctx context.Context, clusterScope *scope.Cluster
 func deleteSecurityGroup(ctx context.Context, clusterScope *scope.ClusterScope, securityGroupId string, securityGroupSvc security.OscSecurityGroupInterface, clock_time clock.Clock) (reconcile.Result, error) {
 	clusterScope.Info("Check loadbalancer deletion")
 
-	currentTimeout := clock_time.Now().Add(time.Second * 20)
+	currentTimeout := clock_time.Now().Add(time.Second * 600)
```

Comment:
one question, how did you end up choosing this numbers ? (600 rather than 20 for example? )

Reply:
This is just a remark but rather than setting timeouts, is it possible in case of failure to just try in the next reconciliation loop ?

Reply:
We can set reconcile.Result{RequeueAfter: 30 * time.Second}

https://github.com/outscale-dev/cluster-api-provider-outscale/blob/ebe14a6c11a6cd797dc8a66d0911b6e2f2c8edcf/controllers/oscmachinetemplate_capacity_controller.go#L78

Reply:
It will be looked during this refacto https://github.com/outscale-dev/cluster-api-provider-outscale/issues/183